packages/ilmomasiina-components/src/modules/events/index.tsx

```diff
@@ -26,7 +26,7 @@ export function useEventListState({ category }: EventListProps = {}) {
   }, [category]);
 
   return useShallowMemo<State>({
-    events: fetchEvents.result,
+    events: fetchEvents.result ?? [],
```

Comment:
This will neutralize the entire point of the `useShallowMemo` as it will invalidate every time. Move the `[]` to a global constant to keep the reference stable.

Reply:
What does this `useShallowMemo` even try to achieve, and why do we even need it, if we are using `useMemo` anyway?

Reply:
It ensures the caller gets a stable reference (as `ilmomasiina-modules` is public interface). Probably not necessary, true.

Reply:
Actually, why that exists: so that the context isn't unnecessarily updated with the same values if the provider re-renders, forcing all context consumers to re-render (since there's no `useContextWithSelector`)